target/linux/ipq40xx/config-5.4

```diff
@@ -4,22 +4,6 @@ CONFIG_ALIGNMENT_TRAP=y
 CONFIG_AR40XX_PHY=y
 CONFIG_ARCH_32BIT_OFF_T=y
 CONFIG_ARCH_CLOCKSOURCE_DATA=y
-CONFIG_ARCH_HAS_BINFMT_FLAT=y
```

Comment:
Please drop all symbol modifications except the 4019 symbol you need.

Reply:
Done, I separated the kernel refresh and MDIO symbols.